crates/red_knot_module_resolver/Cargo.toml

```diff
@@ -14,6 +14,7 @@ license = { workspace = true }
 ruff_db = { workspace = true }
 ruff_python_stdlib = { workspace = true }
 
+clap = { workspace = true }
```

Comment:
Hmm, this is the part that annoys me about clap. It shouldn't be necessary to have a cli dependency (and such a heavy one) on non cli crates. 

Ideally we have the same enum in red knot (we'll need it for other things as well). Alternatively, add a clap feature and make it an optional dependency